### DISCUSSION TITLE
Red Hat Enterprise Linux

### DISCUSSION THREAD

**Discussion by 15ah35** [inventree/InvenTree#5458] [Q&A]
Can I deploy Inventree on Red Hat Enterprise Linux?

**Comment by matmair** [comment]
I have heard of that occasionally but no one in the core team has that deployment. If you know your way around RHEL it should not be a problem while following the bare metal instructions. If you are allowed and do not need a highly efficient instance I would recommend using docker though - a lot simpler.
